test/test_helper.exs

```diff
@@ -17,11 +17,14 @@ defmodule TimeHelper do
 end
 
 defmodule TestHelper do
+  @fcm_http_port 4001
   def reload_app() do
     Application.stop(:mongoose_push)
+    Application.stop(:sparrow)
     Application.unload(:mongoose_push)
     Application.load(:mongoose_push)
     {:ok, _} = Application.ensure_all_started(:mongoose_push)
+    Application.put_env(:goth, :endpoint, "http://localhost:#{@fcm_http_port}")
```

Comment:
Let's put that before we start `:mongoose_push` to make sure it's loaded, as `:goth` may in future change so that for example it reads config only on startup.